traffic_ops/ort/atstccfg/toreq/caching.go

```diff
@@ -84,7 +84,12 @@ func WriteCacheJSON(tempDir string, cacheFileName string, obj interface{}) {
 	}
 	defer objFile.Close()
 
-	if _, err := objFile.Write(objBts); err != nil {
+	if err := objFile.Truncate(0); err != nil { // must truncate, or Write will leave whatever existed after the beginning, if the file already existed and was longer than our new write.
```

Comment:
Could we just add the `os.O_TRUNC` flag to `os.OpenFile` above? That would be more straightforward than manually truncating the file after opening and writing specifically to offset 0.

Reply:
Changed.